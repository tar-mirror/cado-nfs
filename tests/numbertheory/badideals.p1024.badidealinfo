# bad ideals for poly1=-1385-223*x-348*x^2+531*x^3+440*x^4+1090*x^5+1155*x^6
# p=2, r=1 : 2 ideals among 3 are bad
# I1:=ideal<O|2,(5+2668050*alpha^2+1540798875*alpha^3+5338868101875*alpha^4+2055464219221875*alpha^5)/2>; // f=1 e=1
# I2:=ideal<O|2,(7+2310*alpha+1540798875*alpha^3+5338868101875*alpha^4+2055464219221875*alpha^5)/2>; // f=1 e=2
# 1 branch found
2 1 1 1 -1 1
# p=3, r=1 : 2 ideals among 5 are bad
# I2:=ideal<O|3,(8+3465*alpha+889350*alpha^2+342399750*alpha^3+285618458125*alpha^4+25376101471875*alpha^5)/3>; // f=1 e=1
# I3:=ideal<O|3,(20+8085*alpha+2223375*alpha^2+342399750*alpha^3+285618458125*alpha^4+25376101471875*alpha^5)/3>; // f=1 e=1
# 3 branches found
3 2 1 1 1 1
3 2 4 1 -1 1
3 2 7 1 1 -1
