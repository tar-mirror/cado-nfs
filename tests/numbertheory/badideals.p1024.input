-ell 1120320311183071261988433674300182306029096710473 -poly badideals.p1024.poly -badideals badideals.p1024.badideals -badidealinfo badideals.p1024.badidealinfo
