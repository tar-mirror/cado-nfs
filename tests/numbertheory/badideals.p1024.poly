n: 163323987240443679101402070093049155030989439806917519173580070791569227728932850358498862854399351423733697660534800194492724828721314980248259450358792069235991826588944200440687094136669506349093691768902440555341493237296555254247379422702221515929837629813600812082006124038089463610239236157651252180491
skew: 1.0
poly0: -663612177378148694314176730818181556491705934826717,567162312818120432489991568785626986771201829237408
poly1: -1385,-223,-348,531,440,1090,1155
# poly0 = 567162312818120432489991568785626986771201829237408*x-663612177378148694314176730818181556491705934826717
# poly1 = 1155*x^6+1090*x^5+440*x^4+531*x^3-348*x^2-223*x-1385
