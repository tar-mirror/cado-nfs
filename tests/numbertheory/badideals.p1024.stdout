# nmaps0 0
# nmaps1 3
